platform-sdk/swirlds-jasperdb/src/main/java/com/swirlds/merkledb/files/DataFileMetadata.java

```diff
@@ -35,63 +42,56 @@
  * DataFile's metadata that is stored in the data file's footer
  */
 @SuppressWarnings("unused")
-public final class DataFileMetadata {
-    /**
-     * The file format version, this is ready in case we need to change file format and support
-     * multiple versions.
-     */
-    private final int fileFormatVersion;
-    /**
-     * The data item value's size, if the file contains fixed size data items then this is the size
-     * in bytes of those items. If the file contains variable size items then this is the constant
-     * VARIABLE_DATA_SIZE.
-     */
-    private final int dataItemValueSize;
+// Future work: make this class final, once DataFileMetadataJdb is dropped
```

Comment:
Please create a ticket for this future work and leave references to the ticket here (at least, one per class needs to be changed) to make it easier to track the files to be changed.

Reply:
OK, I filed the following ticket about dropping JDB format in the future: https://github.com/hashgraph/hedera-services/issues/8344 - and updated comments with the link